### DISCUSSION TITLE
Need help around the react project from which the minified code of openChat's "backend-server/public/chat.js" is generated from

### DISCUSSION THREAD

**Discussion by tk1994** [opencx-labs/copilot#470] [General]
We want to add more features to OpenChatAi's bot chat UI page. Need help to find the react project from which the minified code of openChat's "backend-server/public/chat.js" is generated from. The chat.js seems to be the file responsible for rendering the UI and we cant find the source from where it is built.

**Comment by faltawy** [comment]
you mean the chat widget ? 
if so you can find the source code inside the `copilot-widget` directory

**Reply by Verma800** [reply]
<img width="1440" alt="Screenshot 2024-01-16 at 11 43 36 AM" src="https://github.com/openchatai/OpenCopilot/assets/113543370/0f55356f-2686-48dd-a499-7b871c79d136">
this file is minified and controlls a lot of part of chatbot.  we wanted to find from which project this file is generated from.
